src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -46,6 +47,7 @@ public class ClusterStateMachine : RachisStateMachine
         private static readonly TableSchema ItemsSchema;
         private static readonly Slice EtagIndexName;
         private static readonly Slice Items;
+        private static readonly ConcurrentSet<string> OngoingDatabaseRestores = new ConcurrentSet<string>();
```

Comment:
No, this is a bad idea. You aren't actually safe since the user might be doing the restore on one machine and adding this on another. We should ignore this entirely since this is something that the admin is doing explicitly and it is rare.